### PR TITLE
Create a suggested solution for fixing the IdP sidebar issue

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** A new set of social IdP pages were published, the landing page of which can be seen at https://developer.okta.com/docs/guides/identity-providers/. Whilst these new pages look great, there is an overriding problem — when you click on one of the cards to access a social IdP/external IdP page, the nav bar breaks. It is no longer shown expanded, and the reader can't see where they are in the site. This PR is a suggested short term fix for this issue, while we explore the problem more.  
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js [hunk 1]
```diff
@@ -175,9 +175,93 @@ export const guides = [
...
+                  {
+                    title: "LinkedIn",
+                    path: "/docs/guides/add-an-external-idp/linkedin/main/"
+                  },
+                  {
+                    title: "Microsoft",
+                    path: "/docs/guides/add-an-external-idp/microsoft/main/"
+                  },
+                  {
+                    title: "OpenID Connect",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/const/navbar.const.js:220]: take this one out and leave the dup in the Enterprise section......

**Review by susanharper-okta — COMMENTED** [review]
one comment

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js (hunk 1 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-theme-prose/const/navbar.const.js]: Fixed! Thanks @susanharper-okta. With this removed, are you OK to give this one your blessing now?

**Review by susanharper-okta — APPROVED** [review]
LGTM - just be wary of merge conflicts with our other issue.

**Pull request merged**